third_party/move/move-vm/paranoid-tests/tests/type_safety/struct/exists.mvir

```diff
@@ -27,7 +27,7 @@ label b0:
     return;
 }
 
-//# run --signers 0x1 
```

Comment:
nit: same as before w. space

Reply:
I think this should no longer be in the diff?